frontend/src/pages/GranteeRecord/components/GranteeSummary.js

```diff
@@ -2,23 +2,39 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Container from '../../../components/Container';
 import './GranteeSummary.css';
+import { getDistinctSortedArray } from '../../../utils';
 
-function GranteeInformationSection({ heading, grants, property }) {
+function GranteeInformationSection({
+  heading, grants, property, distinct,
+}) {
+  let distinctList;
```

Comment:
If we change this line to

```
let distinctList = grants;
```
Seems like we could drop the if down on line 18.

Reply:
It wasn't as easy as setting the one line to grants, but I think I have cleaned this up a bit.